torch_frame/data/multi_nested_tensor.py

```diff
@@ -43,12 +43,25 @@ def from_tensor_mat(
         :obj:`tensor_mat`.
 
         Args:
-            tensor_mat List[List[Tensor]]: A dictionary of
-                matrix of PyTorch Tensors. :obj:`tensor_mat[i][j]` contains
-                1-dim PyTorch Tensor of :obj:`i`-th row and :obj:`j`-th column.
+            tensor_mat (List[List[Tensor]]): A matrix of :class:`torch.Tensor`.
+                :obj:`tensor_mat[i][j]` contains 1-dim PyTorch Tensor of
+                :obj:`i`-th row and :obj:`j`-th column.
 
         Returns:
-            MultiNestedTensor: Returned the class object.
+            MultiNestedTensor: A :class:`MultiNestedTensor` instance.
+
+        Example:
+            >>> tensor_mat = [
+            ...    [torch.tensor([1, 2, 3]), torch.tensor([4, 5])],
+            ...    [torch.tensor([6, 7]), torch.tensor([8, 9, 10])],
+            ... ]
+            >>> out = MultiNestedTensor.from_tensor_mat(tensor_mat)
+            >>> out
+            MultiNestedTensor(num_rows=2, num_cols=2, device='cpu')
+            >>> out.values
+            tensor([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10])
+            >>> out.offset
+            tensor([ 0,  3,  5,  7, 10])
```

Comment:
Shal we add something like
```python
>>> out[0, 1]
>>> tensor_mat[0, 1]
```
to show they are the same?

Reply:
Done!